### ISSUE TITLE
Question about directory structure

### ISSUE THREAD

**Issue by sidarthd** [mysee1989/TCAE#10]
'data/id09238/VqEJCd7pbgQ/0_folder/img_0_001.jpg'

What is '0_folder' here? Is it the frame number? If so how are you sampling faces of the same person from the video?

**Comment by mysee1989** [comment]
0_folder is the directory that I created to store  facial images extracted from a single video clip.

Since each folder only contains frames/images extracted from a single video clip, then it is convenient to sample arbitrary facial image pairs.

**Comment by sidarthd** [comment]
How did you get the timestamps and durations of clips in a video?

**Comment by mysee1989** [comment]
During training, there is no need to get the timestamps or durations of clips in a video.

Before training, I extracted and save the facial frames at 1 fps from each video. 
Each face was detected & aligned.

**Comment by sidarthd** [comment]
Okay, did you save all of the faces in one video in a single folder ?

**Comment by mysee1989** [comment]
Yes.  The facial frames from a video should be saved in a single folder.

**Comment by enginbaglayici** [comment]
Hi, 

I am also trying to understand directory structure but I couldn't figure out why there are 4 folders ('data/id09238/VqEJCd7pbgQ/0_folder/img_0_001.jpg') 
As far as I understand:
**id09238** means a subject.
**VqEJCd7pbgQ** - that I couldnt understand
**0_folder** - belongs to a video of the subject

Thanks for the help :)
